addons/account/models/account_move.py

```diff
@@ -340,6 +340,11 @@ def _sequence_year_range_monthly_regex(self):
         copy=False,
         domain=[('display_type', 'in', ('product', 'line_section', 'line_note'))],
     )
+    invoice_sent_status = fields.Selection(
+        selection=[('sent', 'Sent'), ('not_sent', 'Not sent')],
+        compute="_compute_is_sent",
+        copy=False
+    )
```

Comment:
Rule of thumbs in python, use double quotes when it's something the user will see otherwise use single quotes
```suggestion
    invoice_sent_status = fields.Selection(
        selection=[
            ('sent', "Sent"),
            ('not_sent', "Not sent"),
         ],
        compute='_compute_is_sent',
        copy=False
    )
```